Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[workspace]
+members = [
+  "frb_codegen",
+  "frb_rust",
+  "frb_macros",
+  "frb_example/pure_dart/rust",
+  "frb_example/pure_dart_multi/rust",
+  "frb_example/with_flutter/rust",
+]
+# TODO: refactor with glob see https://github.com/rust-lang/cargo/issues/6009
+# used to ignore corrosion checkout files in CI tests (Linux / Windows)
+exclude = [
+  "corrosion/test/cpp2rust/rust",
+  "corrosion/test/rust2cpp/rust",
+  "corrosion/test/rustflags/rust",
+  "corrosion/test/nostd/rust",
+  "corrosion/test/hostbuild",
+  "corrosion/test/envvar",
+  "corrosion/test/features/rust",
+  "corrosion/test/gensource/srcgen",
+  "corrosion/test/gensource/generated",
+  "corrosion/test/multitarget",
+  "corrosion/test/gensource/generated",
+  "corrosion/test/workspace/member1",
+  "corrosion/test/workspace/member2",
+  "corrosion/test/workspace/member3",
+  "corrosion/generator",
+  "corrosion/test/cargo_flags/rust",
+  "corrosion/test/custom_profiles/rust",
+]
+
+[workspace.package]
+version = "1.48.0"
+edition = "2018"
+license = "MIT"
+description = "High-level memory-safe binding generator for Flutter/Dart <-> Rust"
+repository = "https://github.com/fzyzcjy/flutter_rust_bridge"
+keywords = ["flutter", "dart", "ffi", "code-generation", "bindings"]
+categories = ["development-tools::ffi"]
+
+[workspace.dependencies]
+anyhow = "1.0.64"
+chrono = "=0.4.20"
+lazy_static = "1.4.0"
+uuid = "1.1.2"
+
+flutter_rust_bridge_macros = { path = "frb_macros", version = "1" }
+flutter_rust_bridge = { path = "frb_rust", version = "1" }
+flutter_rust_bridge_codegen = { path = "frb_codegen", version = "1" }
```

Comment:
what I'm not a big fan of is this though, as I can foresee issue when e.g. you will try to publish to crates.io since the CI might end up trying to generate a release for a version `1`. IMHO it should use `workspace.package`'s `version` as it is `sed` with `just release` in another workflow job.

Reply:
what about it @Desdaemon @fzyzcjy ?

Reply:
Not really get it... Where will the "1" be used?

Reply:
I would prefer to explicitly set e.g. `1.48.0` instead of `1`, to be 100% sure to not end up having the wrong version pinned during CI or scripts execution I mean @fzyzcjy :)

Reply:
That LGTM, and would be great if no duplication

Reply:
That LGTM, and would be great if no duplication :)

you mean in the current state with `1` or rewinded with `1.48.0` ? @fzyzcjy

Reply:
Indeed I am not sure. Maybe you and @Desdaemon decide, as long as the release script works :)

Reply:
pinning a specific version will always work @fzyzcjy
on the other hand, pinning a major works ... _most of the time_ I'd say 😅